packages/loot-design/src/components/modals/ImportTransactions.js

```diff
@@ -508,7 +528,7 @@ export function ImportTransactions({
   let [fieldMappings, setFieldMappings] = useState(null);
   let [splitMode, setSplitMode] = useState(false);
   let [flipAmount, setFlipAmount] = useState(false);
-  let { accountId, onImported } = options;
+  let { accountId, categories, onImported } = options;
```

Comment:
Is this variable used somewhere?
```suggestion
  let { accountId, onImported } = options;
```